src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const rules = {};
+
+  sourceString
+    .trim()
+    .split(';')
+    .forEach((rule) => {
+      const [key, value] = rule
+        .trim()
+        .split(':')
+        .map((part) => part.trim());
+
+      rules[key] = value;
```

Comment:
[CODE KNOWLEDGE] - You should check if the key is not an empty string before adding it to the `rules` object to handle cases where there are multiple semicolons without any styles between them. Although it is not required to handle multiple semicolons, it's good practice to ensure that only valid CSS declarations are added to the object.